### ISSUE TITLE
Inappropriate Screen reader text for default/single variant Select

### ISSUE THREAD

**Issue by leducquan** [cerner/terra-core#2742]
# Bug Report

## Description
We are using Terra Select for our DropDown menu component. When tabbing to the component, the Jaws screen reader reads something along the lines of "Search {10 minutes} edit combo collapse. List of {23} options...". We are using the default Form Select which does not allow editing. We don't think it should read out "edit combobox collapse". 

It's also happening on the main Terra UI documentation https://engineering.cerner.com/terra-ui/components/terra-form-select/form-select/select

## Steps to Reproduce
1. Open Jaws Screen Reader.
2. Tab to default/single varient Select

## Additional Context / Screenshots
n/a

## Expected Behavior
We don't expect edit combo collapse to be read, Instead a more appropriate text.

## Possible Solution
More appropriate text. Designer consultation.

## Environment
Jaws 2020
Windows 10

## @ Mentions
n/a

**Comment by scottwilmarth** [comment]
@StephenEsser , my apologies Emily Ogle and I have not had a chance to review this yet. Has WAI-ARIA best practices page been referred to yet? If not, I suggest start there: https://www.w3.org/TR/wai-aria-practices/#combobox

**Comment by jeremyfuksa** [comment]
@leducquan I'm not getting JAWS to reproduce your bug example. Using JAWS 2020.1910.54 in IE11 and IE10, I hear nothing about collapse.

I'm hearing, "Search Blue combo box, list of 5 options..."

**Comment by leducquan** [comment]
@jeremyfuksa In Chrome it reads combo collapse. We also don't believe it should read combobox for a single variant drop down.

**Comment by jeremyfuksa** [comment]
> @jeremyfuksa In Chrome it reads combo collapse. We also don't believe it should read combobox for a single variant drop down.

I'll review with Chrome and see what I get. As a point of discussion, what does your team think that JAWS should say for this component? Combobox would be appropriate for this type of UI component, e.g. https://www.jqwidgets.com/react/react-combobox/index.htm

**Comment by leducquan** [comment]
Combobox is only appropriate when information can be input in the text box and a specific item can be searched for.

For the single variant Select, maybe https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

**Comment by scottwilmarth** [comment]
I reviewed the new select with options using VoiceOver on Chrome and Safari. A few notes:

- With Chrome I can get focus on select element, open with space bar, but I cannot effectively use the dropdown menu using arrow keys because on first arrow down, the first option gets selected and the dropdown closes with no other user interaction.

- With Safari, I can effectively navigate and select via the menu as a keyboard user.

- A couple of WCAG requirements surrounding the dropdown menu opening: First is **Focus Reveal** where the user should be autofocused to the new content or within one click of navigating into the new content. This requiremen is met. The second is **Focus Return**. When the user closes the new content (here wheter it is an ESC or a slection) the user focus is returned to the place new content originated (the select in this case) or the next logical place in the flow of the document (generally the next tab forward). For the current implementation it looks like the user focus is where it should be on close, but on a SHIFT + TAB (backwards) or TAB (move forward) the user focus is moved completely out of context.
-- In Chrome using either ESC or item selection fails Focus Return
-- In Safari closing via ESC fails Focus Return

I will test with JAWS next.

**Comment by scottwilmarth** [comment]
In testing the new select with options with JAWS here are some findings:
- Chrome: fails Focus Return using Escape
- Edge: 
-- Fails Focus Visible (can not tell what item is selected when using keyboard)
-- Fails Focus Return using Escape
-- User can select the option header (not expected)
- IE 11:
-- Fails Focus Return using Escape
- Focus Return fails many of the browers when user clicks tab and the dropdown menu is open.
